ddtrace/tracer/transport.go

```diff
@@ -105,11 +103,40 @@ func newHTTPTransport(addr string, client *http.Client) *httpTransport {
 	}
 	return &httpTransport{
 		traceURL: fmt.Sprintf("http://%s/v0.4/traces", resolveAddr(addr)),
+		statsURL: fmt.Sprintf("http://%s/v0.6/stats", resolveAddr(addr)),
 		client:   client,
 		headers:  defaultHeaders,
 	}
 }
 
+func (t *httpTransport) sendStats(p *statsPayload) error {
+	var buf bytes.Buffer
+	if err := msgp.Encode(&buf, p); err != nil {
+		return err
+	}
+	req, err := http.NewRequest("POST", t.statsURL, &buf)
+	if err != nil {
+		return err
+	}
+	resp, err := t.client.Do(req)
```

Comment:
do we want to add retries here?

Reply:
Maybe? Might be a good idea. If we do, we should do it in a separate PR for both traces and stats. We currently don't have it for traces either.